po/pt_BR.po

```diff
@@ -1,141 +1,142 @@
-# Brazilian portuguese translation for pdfarranger.
+# Brazilian portuguese translation for PDF Arranger.
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the PACKAGE package.
 # Rogênio Belém <rogeniobelem@gmail.com>, 2013
-# Túlio César do Nascimento Peroni <tulioperoni@outlook.com>; 2020
+# Túlio César do Nascimento Peroni <tulioperoni@outlook.com>; 2021
+#
+#, fuzzy
 msgid ""
 msgstr ""
 "Project-Id-Version: pdfarranger\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2021-01-01 22:06+0100\n"
-"PO-Revision-Date: 2021-01-10 11:09-0300\n"
+"POT-Creation-Date: 2021-10-31 13:42+0100\n"
+"PO-Revision-Date: 2021-11-01 15:25-0300\n"
 "Last-Translator: Túlio Peroni <tulioperoni@outlook.com>\n"
-"Language-Team: BR\n"
+"Language-Team: BR (Brazil)\n"
 "Language: pt_BR\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=2; plural=n != 1;\n"
-"X-Generator: Poedit 2.4.2\n"
 
-#: pdfarranger/core.py:80
+#: pdfarranger/core.py:85
 msgid "page"
 msgstr "página"
 
-#: pdfarranger/core.py:175
+#: pdfarranger/core.py:193
 msgid "Password required"
 msgstr "Senha obrigatória"
 
-#: pdfarranger/core.py:186
+#: pdfarranger/core.py:204
 msgid "The password will be remembered until you close PDF Arranger."
 msgstr "A senha será lembrada até que você feche o PDF Arranger."
 
-#: pdfarranger/core.py:187
+#: pdfarranger/core.py:205
 msgid ""
+#, python-format
```

Comment:
According to CI this is an error. Try moving it up one line, so it doesn't stand between msgid and the following string. Did you write this by hand or was this error introduced by some tooling?

Reply:
Sorry, I made a mistake while correcting this line. I accidentally reversed the lines.

Reply:
Don't be sorry, I just wanted to know whether that was a bug somwhere else that needs to be fixed. Thanks for your translations & PR!